pkg/cloudprovider/providers/photon/photon_test.go

```diff
@@ -140,7 +140,7 @@ func TestInstances(t *testing.T) {
 	}
 	t.Logf("Found InstanceID(%s) = %s\n", testVM, instanceId)
 
-	instanceId, err = i.InstanceID(context.TODO(), nonExistingVM)
+	_, err = i.InstanceID(context.TODO(), nonExistingVM)
```

Comment:
instead of removing this here, maybe we could use it in the error message? wdyt?

Reply:
oh, I think this is ok to removed (as in https://github.com/kubernetes/kubernetes/pull/66304/files#diff-3c386806c8805a8326bbd408293f552eR360).